graylog2-web-interface/src/views/components/ViewActionsMenu.tsx

```diff
@@ -65,23 +68,35 @@ const ViewActionsMenu = ({ view, isNewView, metadata }) => {
     </>
   );
 
+  const showSaveButton = viewActionsLayoutOptions === ViewActionsLayoutOptions.FULL_MENU;
+  const showSaveAsButton = viewActionsLayoutOptions === ViewActionsLayoutOptions.FULL_MENU || viewActionsLayoutOptions === ViewActionsLayoutOptions.SAVE_COPY;
+  const showShareButton = viewActionsLayoutOptions === ViewActionsLayoutOptions.FULL_MENU;
+  const showDropDownButton = viewActionsLayoutOptions === ViewActionsLayoutOptions.FULL_MENU;
+
   return (
     <ButtonGroup>
+      {showSaveButton && (
       <Button onClick={() => onSaveView(view)}
               disabled={isNewView || hasUndeclaredParameters || !allowedToEdit}
-              data-testid="dashboard-save-button">
+              title="Save Button">
```

Comment:
What du you think about a title like `Save dashboard` and `Save as new dashboard` instead of `Save Button` and `Save As Button`? The title of the component is `ViewActionsMenu`, but we only use it for dashboards.

Reply:
Fixed.